src/xdocs/config_filters.xml

```diff
@@ -991,6 +991,216 @@ public class InputTest {
+          Checks violations in the specified file, class, method or variable.
+        </p>
+        <p>
+            Attention: This filter only supports single suppression, and will need
+            multiple instances if users wants to suppress multiple violations.
+        </p>
+      </subsection>
+      <subsection name="Properties" id="SuppressionXpathSingleFilter_Properties">
+        <table>
+            <tr>
+                <th>name</th>
+                <th>description</th>
+                <th>type</th>
+                <th>default value</th>
+                <th>since</th>
+            </tr>
+            <tr>
+                <td>files</td>
+                <td>a Regular Expression matched against the file name associated
+                    with an audit event. It is optional.</td>
+                <td><a href="property_types.html#regexp">Regular Expression</a></td>
+                <td><code>null</code></td>
+                <td>8.16</td>
+            </tr>
+            <tr>
+                <td>checks</td>
+                <td>a Regular Expression matched against the name of the check
+                    associated with an audit event. Optional as long as id or message
+                    is specified.</td>
+                <td><a href="property_types.html#regexp">Regular Expression</a></td>
+                <td><code>null</code></td>
+                <td>8.16</td>
+            </tr>
+            <tr>
+                <td>message</td>
+                <td>a Regular Expression matched against the message of the check associated
+                    with an audit event. Optional as long as checks or id is specified.</td>
+                <td><a href="property_types.html#regexp">Regular Expression</a></td>
+                <td><code>null</code></td>
+                <td>8.16</td>
+            </tr>
+            <tr>
+                <td>id</td>
+                <td>a string matched against the ID of the check associated with an audit event.
+                    Optional as long as checks or message is specified.</td>
+                <td><a href="property_types.html#string">String</a></td>
+                <td><code>null</code></td>
+                <td>8.16</td>
+            </tr>
+            <tr>
+                <td>query</td>
+                <td>a string xpath query. It is optional.</td>
+                <td><a href="property_types.html#string">String</a></td>
+                <td><code>null</code></td>
+                <td>8.16</td>
+            </tr>
+        </table>
+      </subsection>
+      <subsection name="Examples" id="SuppressionXpathSingleFilter_Examples">
+        <p>
+          To configure to reject CyclomaticComplexity errors
+          for all methods with name sayHelloWorld
+          inside FileOne ane FileTwo classes:
+        </p>
+        <pre>
+&lt;module name="SuppressionXpathSingleFilter"&gt;
+  &lt;property name="files" value="FileOne.java, FileTwo.java"/&gt;
```

Comment:
`files` isn't a comma separated field, it is a regular expression. Please fix this example.
I would recommend testing your examples to make sure they are valid if you are unsure.

`inside FileOne ane FileTwo classes` => `inside FileOne ane FileTwo files`.

I see you copied this from somewhere else, but it is wrong too. Please update it too.

Reply:
I am not sure how to "test" my example. The way I learned to test the example is just use the jar of checkstyle to test on the command line. But now my changes didn't involved inside the jar of checkstyle, so I don't know how could I test it on "two" files. Or should I add this inside the testing file?

Reply:
your Filer is inside of jar, all you need is to create this jar on your local: `mvn package -Passembly` and recheck your `target` folder. Use this jar to do testing the same as with released version(that you download from web).

Reply:
done. See the test below.